<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmployeeWindow</class>
 <widget class="QDialog" name="AddEmployeeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление сотрудника</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/add.png</normaloff>../icons/add.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="2">
    <widget class="QLineEdit" name="lineFirstName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Введите имя...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="linePatronymic">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Введите отчество...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QComboBox" name="comboRole">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string>Выберите должность...</string>
     </property>
     <item>
      <property name="text">
       <string>Администратор</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сотрудник</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="lineSurname">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Введите фамилию...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLineEdit" name="lineLogin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="placeholderText">
      <string>Введите логин...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Убедитесь, что все данные введены верно</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLineEdit" name="linePassword">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Введите пароль...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineFirstName</tabstop>
  <tabstop>lineSurname</tabstop>
  <tabstop>linePatronymic</tabstop>
  <tabstop>lineLogin</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>comboRole</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEmployeeWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEmployeeWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
